### PR TITLE
[fix](exit) stop storage engine in exec_env before exit

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

File: be/src/runtime/exec_env_init.cpp [hunk 1]
```diff
@@ -675,6 +675,7 @@ void ExecEnv::destroy() {
 
     // StorageEngine must be destoried before _page_no_cache_mem_tracker.reset and _cache_manager destory
     // shouldn't use SAFE_STOP. otherwise will lead to twice stop.
+    SAFE_STOP(_storage_engine);
```
Comment (gavinchou) [inline be/src/runtime/exec_env_init.cpp:678]: the comment says cannot invoke SAFE_STOP for storage engein

**Comment by dataroaring** [comment]
run buildall

**Review by Yukang-Lian — APPROVED** [review]

**Review by w41ter — APPROVED** [review]

**Pull request merged**